### ISSUE TITLE
App crashes at start, `userVersion != currentUserVersion` in Postbox.swift

### ISSUE THREAD

**Issue by bitemyapp** [overtake/TelegramSwift#161]
<img width="884" alt="screen shot 2018-05-05 at 2 05 35 pm" src="https://user-images.githubusercontent.com/320177/39666717-e9728a30-506d-11e8-9771-45b9737f4685.png">

I think userVersion is 16?

<img width="872" alt="screen shot 2018-05-05 at 2 09 50 pm" src="https://user-images.githubusercontent.com/320177/39666733-ffe99808-506d-11e8-96ce-20af84690599.png">

I believe this is my checkout:

```
commit e04d88acdbc827827da0c06ccde3cee2dcff2743 (HEAD -> master, origin/master, origin/HEAD)
Merge: 2e30c58 8ba3825
Author: Mike <overtakeful@gmail.com>
Date:   Fri Dec 29 14:22:27 2017 +0400

    Merge pull request #64 from iSevenDays/master
    
    Implement scroll for InstantView using space/shift space key combinations
```

**Comment by bitemyapp** [comment]
It worked after I bumped currentUserVersion to 16, but I'm not sure if that was the right thing to do.

I also got an update notice when I started the local dev version of the app. Is this repository not up to date with the current release?

<img width="594" alt="screen shot 2018-05-05 at 2 12 19 pm" src="https://user-images.githubusercontent.com/320177/39666760-80c0c32a-506e-11e8-8bd0-880cde4b1fc8.png">
